src/db/entities/coinPrice.ts

```diff
@@ -0,0 +1,14 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { Coin } from './coin';
+
+@Entity()
+export class CoinPrice {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column(() => Coin)
```

Comment:
You are doing a good thing here, but maybe for the sake of easier search, just put 'coinId" or something like that. A simple integer...

Reply:
sure, done.